ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/sdn2ovn/provision/cucushift-installer-rehearse-gcp-ipi-ovn-sdn2ovn-provision-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-ovn-sdn2ovn-provision
+  steps:
+  - chain: ipi-conf-gcp
```

Comment:
@weliang1 There is an issue now for 4.12
The default network type will be OVN in 4.12. So here we need to make sure the cluster is sdn 
maybe we can add 

```
- ref: sdn-conf
```

Reply:
@zhaozhanqi This PR is still for openshift-verification-tests-master__installer-rehearse-4.11.yaml. Let's merge this first. Then we can update sdn-conf in all sdn2ovn workflows for 4.12 release. HDYT?

Reply:
workflow is not only for one version, any config job can invoke the workflow. 
and `openshift-verification-tests-master__installer-rehearse-4.11.yaml` this file is only for testing the workflow is correct.  
it's not in real CI.

Reply:
> workflow is not only for one version, and each version config job can invoke the workflow. eg. this may works in 4.11 config file if invoke this workflow. `./ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml`
> 
> but will failed in `ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml`
> 
> and `openshift-verification-tests-master__installer-rehearse-4.11.yaml` this file is only for testing the workflow is correct. it's not in real CI.
> 
> and another things is we also need to update the exiting sdn2ovn workflow to make sure it's SDN plugin.

@zhaozhanqi Thanks for good explanations!  Add `ref: sdn-conf` in workflow now.